app/src/main/java/com/sopt/clody/presentation/ui/setting/navigation/SettingNavigator.kt

```diff
@@ -13,6 +14,11 @@ class SettingNavigator(
         navController.navigate("notification_setting")
     }
 
+    fun navigateWebView(url: String) {
+        val encodedUrl = URLEncoder.encode(url, "UTF-8")
```

Comment:
[**P4**]
```suggestion
        val encodedUrl = URLEncoder.encode(url, StandardCharsets.UTF_8.toString())
```
이런 방법도 있음 참 신기하죠?

Reply:
내부코드 까보니까 String으로 받는거랑 동일하게 `encode(String s, Charset charset)`으로도 사용 가능하다고 보이네요 ,, 신기하게도.. 이런건 도대체 언제 까보셨습니까 형님